File: py-codellama34B-AWQ-all/taskid-156_int_to_mini_roman-gen1.py
def int_to_mini_roman(number: int) -> str:
    """
    Given a positive integer, obtain its roman numeral equivalent as a string,
    and return it in lowercase.
    Restrictions: 1 <= num <= 1000

    Examples:
    >>> int_to_mini_roman(19)
    'xix'
    >>> int_to_mini_roman(152)
    'clii'
    >>> int_to_mini_roman(426)
    'cdxxvi'
    """

    roman = ""
    if number >= 500:
        number -= 500
        roman += "d"

    if number >= 100:
        number -= 100
        roman += "c"

    if number >= 50:
        number -= 50
        roman += "l"

    if number >= 10:
        number -= 10
        roman += "x"

    if number >= 5:
        number -= 5
        roman += "v"

    if number >= 1:
        number -= 1
        roman += "i"

    return roman

File: java-codellama7B-AWQ-all/taskid-156_int_to_mini_roman-gen1.java
import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    // Given a positive integer, obtain its roman numeral equivalent as a string,
    // and return it in lowercase.
    // Restrictions: 1 <= num <= 1000
    // Examples:
    // >>> intToMiniRoman((19l))
    // ("xix")
    // >>> intToMiniRoman((152l))
    // ("clii")
    // >>> intToMiniRoman((426l))
    // ("cdxxvi")
    public static String intToMiniRoman(long number) {

        if (number == 0) {
            return "";
        }
        String roman = "";
        int i = (int)number % 10;
        if (i >= 1 && i <= 3) {
            roman += "i";
        }
        if (i == 4) {
            roman += "iv";
        }
        if (i >= 5 && i <= 8) {
            roman += "v";
        }
        if (i == 9) {
            roman += "ix";
        }
        if (number >= 10) {
            roman += "x";
        }
        if (number >= 50) {
            roman += "l";
        }
        if (number >= 100) {
            roman += "c";
        }
        if (number >= 500) {
            roman += "d";
        }
        if (number >= 1000) {
            roman += "m";
        }
        return roman;
    }
    public static void main(String[] args) {
    assert(intToMiniRoman((19l)).equals(("xix")));
    assert(intToMiniRoman((152l)).equals(("clii")));
    assert(intToMiniRoman((251l)).equals(("ccli")));
    assert(intToMiniRoman((426l)).equals(("cdxxvi")));
    assert(intToMiniRoman((500l)).equals(("d")));
    assert(intToMiniRoman((1l)).equals(("i")));
    assert(intToMiniRoman((4l)).equals(("iv")));
    assert(intToMiniRoman((43l)).equals(("xliii")));
    assert(intToMiniRoman((90l)).equals(("xc")));
    assert(intToMiniRoman((94l)).equals(("xciv")));
    assert(intToMiniRoman((532l)).equals(("dxxxii")));
    assert(intToMiniRoman((900l)).equals(("cm")));
    assert(intToMiniRoman((994l)).equals(("cmxciv")));
    assert(intToMiniRoman((1000l)).equals(("m")));
    }

}
